etcd/etcd.go

```diff
@@ -44,7 +44,7 @@ type Client interface {
 	SetParser(ConfigParser)
 	ClientConfigParam(cpc *ConfigParamConfig, cfs ...CustomFunction) (Key, error)
 	ServerConfigParam(cpc *ConfigParamConfig, cfs ...CustomFunction) (Key, error)
-	RegisterConfigCallback(ctx context.Context, key string, clientId int64, callback func(string, ConfigParser))
+	RegisterConfigCallback(ctx context.Context, key string, clientId int64, callback func(bool, string, ConfigParser))
```

Comment:
Name the input parameters of the callback function to make the code easier to understand.